app/views/users/edit.html.erb

```diff
@@ -0,0 +1,28 @@
+<h1>Hello Rails</h1>
+
+<%= form_with model: @user, url: user_path do |form| %>
+
+  <% if @user.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@user.errors.count, "error") %> prohibited this user from being saved:</h2>
+
+      <ul>
+      <% @user.errors.full_messages.each do |msg| %>
+        <li><%= msg %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
```

Comment:
**Optional:**

- [ ] Kindly move the logic to a helper. There should be absolutely NO logic in views. 
Views are only for displaying information that became available to it from its corresponding controller not generating it. Check this [article](http://codefol.io/posts/Where-Do-I-Put-My-Code/) on how to use a helper in rails.